core/transact.go

```diff
@@ -174,13 +174,10 @@ func recordSubmittedTx(ctx context.Context, db pg.DB, txHash bc.Hash, currentHei
 	return height, err
 }
 
-// CleanupSubmittedTxs will periodically delete records of submitted txs
+// cleanupSubmittedTxs will periodically delete records of submitted txs
 // older than a day. This function blocks and only exits when its context
 // is cancelled.
-//
-// TODO(jackson): unexport this and start it in a goroutine in a core.New()
-// function?
-func CleanupSubmittedTxs(ctx context.Context, db pg.DB) {
+func cleanupSubmittedTxs(ctx context.Context, db pg.DB) {
```

Comment:
same nit (as long as this function name is changing anyway):

cleanup = noun
clean up = verb